### PR TITLE
[#799] Add live scene loading

### LINKED ISSUE #799: Add 'live' scene loading
- Add checkbox on `/map` page to display the latest scenes of the product (send a request for new scenes every 15 minutes)
- Reload the latest scene on the map, when new one occur
- When live mode is turn on ask if user want to exit live mode while
  - changing between products
  - trying to change scene (by hour/date or on time axis)

### PR DESCRIPTION
**Pull request by danielkryska**
Where to find feature?

1. Go to `/map`
2. Turn on any of products
3. Click button `Na żywo` near timeline

- Add checkbox on `/map` page to display the latest scenes of the product (send a request for new scenes every 15 minutes)
- Reload the latest scene on the map, when new occur
- When live mode is turn on ask if user is sure while
  - changing between products
  - trying to change scene (by hour/date or on time axis)

Closes #799

### CODE REVIEW

**Comment by michal-szostak** [comment]
With PR's concerning some UI features it's good to assign some screenshots showing how does it look.

**Comment by michal-szostak** [comment]
And I have one question - isn't it better to add this flag to the Scenes store?

**Comment by danielkryska** [comment]
> With PR's concerning some UI features it's good to assign some screenshots showing how does it look.

Unfortunately on my Ubuntu all print screens always are black due to some graphic card issue and there weren't enough time to fix it.

**Review by kmarszalek — APPROVED** [review]
it works, after slack conversation time will be configurable. it reloads after time even if there was no new scene- loads old one.

**Review by michal-szostak — APPROVED** [review]

**Review by kmarszalek — CHANGES_REQUESTED** [review]
commits

File: s4e-web/src/environments/environment.common.ts [hunk 1]
```diff
@@ -5,6 +5,7 @@ export const commonEnvironmentVariables = {
   projection: {toProjection: 'EPSG:3857', coordinates: [19, 52] as [number, number]},
   maxZoom: 12,
   timezone: Intl.DateTimeFormat().resolvedOptions().timeZone,
+  liveSceneUpdateRateInMs: 1000
```
Comment (kmarszalek) [inline s4e-web/src/environments/environment.common.ts:8]: change to more than 1 sek and leave some comment :)

**Review by kmarszalek — CHANGES_REQUESTED** [review]

**Review by kmarszalek — APPROVED** [review]

**Pull request merged**